OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/operations/IOperationRepo.kt

```diff
@@ -42,6 +42,8 @@ interface IOperationRepo {
     suspend fun awaitInitialized()
 
     fun forceExecuteOperations()
+
+    fun setPaused(paused: Boolean)
```

Comment:
I don't see this being called anywhere, if it isn't used we should not include it in this PR.

Reply:
Addressed in the fixup commit